src/interface/api-template.yaml

```diff
@@ -298,3 +352,36 @@ components:
         - consentId
         - sourceAccountId
         - status
+    UpdateThirdpartyAuthorizationRequest:
+      title: UpdateThirdpartyAuthorizationRequest
+      type: object
+      description: The object sent in the PUT /thirdpartyRequests/transactions/{id}/authorizations request.
+      properties:
+        challenge:
+          type: string
+          description: The original Challenge Object as a JSON string
+        value:
+          allOf:
+            - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/schemas/BinaryString.yaml'
+          description: 'Base64 encoded binary string - the signed challenge'
+        consentId:
+          allOf:
+            - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/schemas/CorrelationId.yaml'
+          description: >
+            Common ID between the PISP and FSP for the Consent object
+            This tells DFSP and auth-service which constent allows the PISP to initiate transaction.
+        sourceAccountId:
+          allOf:
+            - $ref: '#/components/schemas/AccountId'
+          description: DFSP specific account identifiers, e.g. `dfspa.alice.1234`
+        status:
+          type: string
+          enum:
+            - VERIFIED
+          description: The status of the authorization. This MUST be VERIFIED for a PUT request
```

Comment:
So I think the logic here is that if there was a verification error, then the `PUT /thirdpartyRequests/transactions/{id}/authorizations/error` callback would be used instead? I think we should just find a similar request in the Mojaloop API spec to double check this pattern.

This is one of the drawbacks with our approach of worrying about the error conditions later on.

Reply:
Ok